build-tests/x86/suse/test-image-qcow-openstack/appliance.kiwi

```diff
@@ -7,11 +7,11 @@
         <specification>SUSE Tumbleweed guest image for OpenStack</specification>
     </description>
     <preferences>
-        <type image="oem" filesystem="ext4" format="qcow2" kernelcmdline="console=tty1 console=ttyS0 net.ifnames=0">
+        <type image="oem" filesystem="ext4" format="qcow2" kernelcmdline="console=ttyS0 net.ifnames=0">
             <oemconfig>
                 <oem-resize>false</oem-resize>
             </oemconfig>
-            <bootloader name="grub2" timeout="1"/>
+            <bootloader name="grub2" timeout="1" console="serial gfxterm"/>
```

Comment:
Any reason to keep the gfxterm here in this test and remove it on others?

Reply:
yes to for testing the possible settings of the grub console parameter. I wanted to have at least one test that uses a multi console setup for grub to make sure the corresponding parts in the kiwi code are reached (e.g the pattern in the schema)

So this one is intentional